std/file.d

```diff
@@ -2832,7 +2832,7 @@ version(Posix) @system unittest // input range of dchars
  * Get the current working directory.
  * Throws: $(D FileException) on error.
  */
-version(Windows) string getcwd()
+version(Windows) string getcwd() @trusted
```

Comment:
this is where this gets a little dangerous. I when through the code and I don't see any security holes. I tried using trusted lambdas here, but the issue is 90% of the function is unsafe.

Reply:
You could use a ScopedAllocator for the allocation + once we fix checked, then most of this should be @safe.